Packs/CommonScripts/TestPlaybooks/IsEmailAddressInternal_Test.yml

```diff
@@ -130,7 +131,7 @@ tasks:
       key:
         simple: emails
       value:
-        simple: '["email@domain.com", "email@sub.test.com", "invalidmail"]'
```

Comment:
You deleted the 'invalidmail' from the Set command, as a result the 'invalidmail' wont be checked in task "IsEmailAddressInternal include_subdomains"